remote-api/pom.xml

```diff
@@ -60,6 +60,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Remove the Lombok from dependency. As mentioned Lombok shall not be used.

Reply:
Done